spec.html

```diff
@@ -900,13 +900,14 @@ <h1>Syntax-Directed Operations</h1>
 
       <emu-clause id="sec-invoking-syntax-directed-operations">
         <h1>Invoking Syntax-Directed Operations</h1>
-        <p>Syntax-directed operations are invoked with a parse node and, optionally, other parameters by using the conventions on steps <emu-xref href="#step-sdo-invocation-example-1"></emu-xref>, <emu-xref href="#step-sdo-invocation-example-2"></emu-xref>, and <emu-xref href="#step-sdo-invocation-example-3"></emu-xref> in the following algorithm:</p>
+        <p>Syntax-directed operations are invoked with a Parse Node and, optionally, other parameters by using the conventions on steps <emu-xref href="#step-sdo-invocation-example-1"></emu-xref>, <emu-xref href="#step-sdo-invocation-example-2"></emu-xref>, and <emu-xref href="#step-sdo-invocation-example-3"></emu-xref> in the following algorithm:</p>
```

Comment:
```suggestion
        <p>Syntax-directed operations are invoked with a subject Parse Node and, optionally, other parameters by using the conventions on steps <emu-xref href="#step-sdo-invocation-example-1"></emu-xref>, <emu-xref href="#step-sdo-invocation-example-2"></emu-xref>, and <emu-xref href="#step-sdo-invocation-example-3"></emu-xref> in the following algorithm:</p>
```

to align with the use of the phrase "subject Parse Node" below

Reply:
I agree with the intent, but I'm worried that a reader might think that "a subject Parse Node" was a reference to a term defined elsewhere. We could put 'subject' in italics or quotes, to convey that this is its defining occurrence (sort of), but I think it'd be clearer to say something like:
> invoked with a Parse Node (referred to below as 'the subject Parse Node')

Also, the sentence was getting a bit complicated, so I broke it into two.

By the way, I'm not sure that "subject" is the best word. In my code, I've also used "focus" and "target".